test/test_configuration.rb

```diff
@@ -12,7 +12,45 @@ def test_change_in_configuration_keep_default_untouched
   end
 
   def test_options
+    hundle_options
     assert @config.options
     assert_includes @config.options, :args
   end
+
+  def test_option_templates
+    hundle_options('--templates')
+    assert(@config.options)
+    assert_includes(@config.options, :templates)
+  end
+
+  def test_option_version
+    hundle_options('-v')
+    assert(@config.options)
+    assert_includes(@config.options, :version)
+  end
+
+  def test_invalid_option
+    assert_throws_and_match(Gem2Rpm::Configuration::ConfigurationError,
+      /\Ainvalid option: -Z\n\nUsage:/m) do
+      hundle_options('-Z')
+    end
+  end
+
+  private
+
+  def hundle_options(string = '')
```

Comment:
Ah, it should be "handle" ... anyway, I'd go with "setup_options" or something alike.